FSJ_django20_project/FSJ/templates/FSJ/comment.html

```diff
@@ -0,0 +1,35 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %} {# NOTE YOU MUST LOAD i18n to every HTML page even though it's also loaded to base.html #}
+
+{% block heading %}
+    {{ application.student.ccid }}
+{% endblock %}
+
+{% block content %}
+    <div class='center-content'>
+        <form action='{{ url }}' method='post'>
```

Comment:
why not include model_form.html like other updates/edits for model forms instead of duplication?

Reply:
is this html page even used?

Reply:
It isn't used, just have not got around to deleting.